test/python/scenarios.json

```diff
@@ -1,4 +1,13 @@
 {
+    "install_python_setuptools_vulnerability": {
+        "image": "debian:bookworm",
+        "features": {
+            "python": {
+                "version": "3.10",
+                "installTools": true
+            }
```

Comment:
```suggestion
        "image": "python:3:10",
        "features": {
            "python": {
                "version": "none",
                "installTools": true
            }
```

Can we update the scenario to reflect the python image build?

Reply:
These are the points that I wanted to discuss with you. The moment we move to version as "none" the code in install.sh will not work.

Reply:
> the code in install.sh will not work.

Which code are you referring at?

Reply:
We have create a separate code set with different branch name and created a PR. Its by Gaurav and the details are as below :
python : https://github.com/devcontainers/features/pull/815
debian : https://github.com/devcontainers/features/pull/814

If Debian work in context with Debian image is not require then we can reject  PR # 814.